1223-curator_worklist/my_worklist.feature

```diff
@@ -115,71 +141,36 @@ Feature: Curator opens My Worklist
 
   @needJiraTag
   Scenario Outline: Curator change Status on a Request
-    When they select a new status "<Status>" on a Request
+    When Curator selects a new status "<Status>" on a Request
     Then the status of the Request is set to "<Status>"
     Examples:
       | Status   |
       | Archived |
       | Deleted  |
 
   @needJiraTag
-  Scenario: Curator claims a Request
-    When the Curator do either:
-      | send an answer        |
-      | Publish               |
-      | Mint a DOI            |
-      | Decline a DOI         |
-      | Change Owner          |
-      | Decline a Ownerchange |
-    Then the Curator is stored as Owner of the Request
-
-  @needJiraTag
-  Scenario: Decide which Approval, Support or DOI Requests is in a Curators Scope
-    When the Requests' Host equals the Curators Host
-    And Curators scope on the Institution include the Requests' Users' User Profile affilliation to the Institution
-    Then the Users Request is displayed to the Curator
-
-
-  @needJiraTag
-  Scenario: Curator opens their Work List
-    When the Curator opens the Work List
-    And it is scoped to an institutional subunit
-    Then the Curator see it is scoped to a subunit
-    And all Requests from Senders in this subunit is listed
-
+  Scenario: Curator is Assigned a Request
+    When the Curator:
+      | sends an answer          |
+      | Publishes a resource     |
+      | Mints a DOI              |
+      | Declines a DOI           |
+      | Changes Owner            |
+      | Declines change of owner |
+    Then the Curator is Assigned the Request
 
   @needJiraTag
-  Scenario: Curator changes scope of Work List
-    When the Curator opens the Work List
-    Then the Curator may change scope to any other subunit at the institution
-    And all Requests from Senders in this new subunit is listed
+  Scenario: Decide if an Approval, Support or DOI Requests is in a Curators Scope
```

Comment:
"Curator receives Requests in their scope and requests they have been assigned from outside their scope", which to me means the latter scenario is to be split out

Reply:
Have duplicated the scenario and adjusted them.